client/src/Header.js

```diff
@@ -33,6 +33,14 @@ const Header = () => {
 						>
 							Grant Access
 						</NavLink>
+						<NavLink
+							exact
+							to="/logout"
```

Comment:
When the user clicks this link, clear `userType` and `authToken` from localStorage and direct the user to `/`